setting/ratio_setting/video_price.go

```
@@ -0,0 +1,116 @@
+// VideoPriceModelPrice 返回按次计费的美元价格:锚点 ¥/秒 → USD per-call。
+func VideoPriceModelPrice(t *VideoPriceTable) float64 {
+	return VideoPriceAnchor(t) / USD2RMB
+}
+
+type OffPeakWindow struct {
+	StartHour int    `json:"start_hour"`
+	EndHour   int    `json:"end_hour"`
+	Timezone  string `json:"timezone"`
+}
+
+const defaultOffPeakWindow = `{"start_hour":22,"end_hour":8,"timezone":"Asia/Shanghai"}`
```

golangci-lint (2.13.2)

[error] const defaultOffPeakWindow is unused

(unused)